tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+| Test suite | A collection of tests that are typically validating a common feature, or should be run together. One test suite can contain many tests and setup or breakdown steps. | Collection of tests, tests | "Let's look at that test suite together, it has 29 passing tests and 18 failing tests," "I have two different test suites: one for my back-end, and one for my front-end," "Let's run that `edit-book-endpoint` test suite and see what tests are failing around the edit book feature" |
+
+## Running JavaScript Tests
+
+When we run JavaScript programs, including running test suites, we need to consider which runtime environment to use.
+
+For this portion of the curriculum, we will run tests using:
+
+- Replit.com
+- Learn code challenges on this curriculum platform
+
+When we have bigger projects with different environments, such as React JS projects, we will run tests on our local machines. For now, we can focus on reading tests and building our skills to pass them.
+
+## Reading Tests Without Learning Syntax
+
+Leila is jumping into a JavaScript project. They're pairing with another developer, Nadia, who has been working on implementing FizzBuzz.
+
+Nadia has already set up the tests for FizzBuzz. Leila wants to jump into coding, but they need to take a moment to read through the tests.
+
+What can Leila determine from reading tests in a syntax they don't know? Let's read through one test and make predictions.
+
+This is in a file named `test/fizzbuzz.test.js`
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  test('returns Fizz on multiples of 3', () => {
+    // Arrange
+    const num = 9;
+
+    // Act
+    const result = fizzBuzz(num);
+
+    // Assert
+    expect(result).toEqual('Fizz');
+  });
+});
+```
+<!-- prettier-ignore-end -->
+
+### The File Itself
+
+First, Leila can quickly orient themself with the file name and where it's located. It seems like there is a `test` directory, and a file named `fizzbuzz.test.js` in it.
+
+From this evidence, they can at least hypothesize this:
+
+- This file is dedicated to validating the correctness of the `fizzBuzz` function
+- It's unlikely that the `fizzBuzz` implementation is in this file, or even this folder
+
+### !callout-info
+
+## Confidently Make Hypotheses
+
+As Leila works through this, they will need to make a lot of assumptions. Leila can take the time to fact-check all of her assumptions, which is a completely valid thing to do (and typically a good strategy for life). However, in this situation, Leila can _also_ confidently rely on their skills, intuition, context, and help from others to make theories about what's going on.
+
+<br/>
+
+"Maybe the `test` folder only holds test files, and we have a `.test.js` file for each function." Making theories like this will help Leila move forward in their learning. It will also help them correct themself and their working knowledge more easily!
```

Comment:
separate the theory and the observation for impact.
```suggestion
"Maybe the `test` folder only holds test files, and we have a `.test.js` file for each function."

<br/>

Making theories like this will help Leila move forward in their learning. It will also help them correct themself and their working knowledge more easily!
```